wicg-mediasession/index.d.ts

```diff
@@ -0,0 +1,58 @@
+// Type definitions for Media Session API 1.0
+// Project: https://wicg.github.io/mediasession/
+// Definitions by: Julien CROUZET <https://github.com/jucrouzet>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Navigator {
+  readonly mediaSession: MediaSession;
```

Comment:
This should be optional right? The sample code [here](https://developers.google.com/web/updates/2017/02/media-session) checks for its existence before using it